docs/user-guide/persistent-volumes/index.md

```diff
@@ -356,6 +356,16 @@ parameters:
 Storage classes have a provisioner that determines what volume plugin is used
 for provisioning PVs. This field must be specified.
 
+Administrators are not restricted to specifying the "internal" provisioners
+listed here (whose names are prefixed with "kubernetes.io" and shipped
+alongside Kubernetes), they can also run and specify external provisioners
```

Comment:
"Administrators" are the readers in this case. Replace with "you." As in "You are not restricted to specifying..."

Replace comma with period, start a new sentence. "You can also run and specify external provisioners, which are independent programs..."